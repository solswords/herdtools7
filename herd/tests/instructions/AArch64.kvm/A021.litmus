AArch64 A021
variant=vmsa,skip
{
0:X0=0;
}
  P0          ;
L0:           ;
 SVC #0       ;
 ADD W0,W0,#1 ;
forall fault(P0:L0) /\ 0:X0=1

