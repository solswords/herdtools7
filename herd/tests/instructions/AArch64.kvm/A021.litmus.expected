Test A021 Required
States 1
0:X0=1; Fault(P0:L0,SupervisorCall);
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (fault(P0:L0) /\ 0:X0=1)
Observation A021 Always 1 0
Hash=8fee4366aa4d7780c51e044180985ce6

